Rakefile

```diff
@@ -54,7 +54,7 @@ namespace :server do
       puts "\r\e[0KStopping ..."
       Rake::Task['server:down'].execute
     end
-    system "bundle exec rerun 'rackup -p $PORT -E $RACK_ENV'"
+    system "bundle exec rerun 'rackup -p $PORT -E $RACK_ENV -o 0.0.0.0'"
```

Comment:
Yes, it works. I have just checked it.

This IP address is a constraint for incoming connections. So, if it is set to 127.0.0.1, then the server will only accept incoming connections from 127.0.0.1.

0.0.0.0 is like a wildcard to allow the incoming connections from anywhere.